### PR TITLE
Add a converter that transforms MD links to HTML links

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**
\cc @MargaritaLoseva @inikulin 
first part of devexpress/testcafe#311

### CODE REVIEW

**Comment by VasilyStrelyaev** [comment]
\cc @DevExpress/testcafe-docs

File: src/_plugins/md-links-to-html-links.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+module ConvertMdLinksToHtml
+    class Generator < Jekyll::Generator
+        def generate(site)
+            site.pages.each { |p| convert_links(site, p) }
+        end
+        def convert_links(site, page)
+            page.content = page.content.gsub(/index\.md/, '')
```
Comment (inikulin) [inline src/_plugins/md-links-to-html-links.rb]: There RE's are too relaxed. I'd rather check that we are in hyperlink.

**Comment by inikulin** [comment]
\r-

**Comment by VasilyStrelyaev** [comment]
Also changed the way we generate anchor ids from headers to comply with github md processor

**Comment by VasilyStrelyaev** [comment]
FPR

**Comment by inikulin** [comment]
lgtm

**Comment by MargaritaLoseva** [comment]
lgtm

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Comment by kirovboris** [comment]
lgtm

**Comment by VasilyStrelyaev** [comment]
ping @churkin

**Comment by churkin** [comment]
lgtm

**Pull request merged**